roles/postgresql_client_access/README.md

```diff
@@ -1,6 +1,6 @@
 # `famedly.base.postgresql_client_access` ansible role for configuring users, databases and pg_hba entries of an PostgreSQL instance
 This convenience role creates and updates users, databases and pg_hba entries.
-It's designed to work with PostgreSQL servers running inside a docker container deployed by [famedly.base.postgresql](https://gitlab.com/famedly/infra/collections/base/-/tree/main/roles/postgresql).
+It's desiged to work with PostgreSQL servers running inside a docker container deployed by [famedly.base.postgresql](https://github.com/famedly/ansible-collection-base/tree/main/roles/postgresql).
```

Comment:
I think you accidentally deleted a character here?

Reply:
Nope, github doesn't have the minus on those urls that gitlab does, so this is correct

Reply:
I meant the "designed" vs "desiged" in the beginning there, not the URLs

Reply:
oh. whoops, sorry about that

Reply:
fixed

Reply:
:+1: